lib/resources/host.rb

```diff
@@ -147,6 +147,69 @@ def missing_requirements(_protocol)
   end
 
   class UnixHostProvider < HostProvider
+    def initialize(inspec)
+      super
+
+      @has_nc = inspec.command('nc').exist?
+      @has_ncat = inspec.command('ncat').exist?
+      @has_net_redirections = inspec.command("strings `which bash` | grep -qE '/dev/(tcp|udp)/'").exit_status == 0
+    end
+
+    def missing_requirements(protocol)
+      missing = []
+
+      if %w{tcp udp}.include?(protocol) and !@has_nc and !@has_ncat
```

Comment:
Can you change the `and`'s to `&&`? We try to avoid and/or due to the lower precedence.

Reply:
Had no idea about the precedence difference, guess I learned something. :) Done.